libs/styles/rounded-corners.css

```diff
@@ -8,24 +8,28 @@
 
 .rounded-corners,
 .rounded-corners > .background img,
+.rounded-corners-image .media-area,
 .rounded-corners-image .foreground .image picture img {
   border-radius: var(--m-rounded-corners);
 }
 
 .s-rounded-corners,
 .s-rounded-corners > .background img,
+.s-rounded-corners-image .media-area,
 .s-rounded-corners-image .foreground .image picture img {
   border-radius: var(--s-rounded-corners);
 }
 
 .m-rounded-corners,
 .m-rounded-corners > .background img,
+.m-rounded-corners-image .media-area,
 .m-rounded-corners-image .foreground .image picture img {
   border-radius: var(--m-rounded-corners);
 }
 
 .l-rounded-corners,
 .l-rounded-corners > .background img,
+.l-rounded-corners-image .media-area,
```

Comment:
I see some issues with border bottom radius
<img width="619" alt="Screenshot 2024-07-12 at 15 00 28" src="https://github.com/user-attachments/assets/0eba523b-db30-4d17-a69f-7e4b620a411d">

Reply:
I saw that and have updated the doc...
This is actually caused by the video first frame rendering in a widescreen format and using the class .`l-rounded-corners-image`. It was probably a poor example. I have updated these use the `l-rounded-corners` instead which targets the block and not the media-area.